README.md

```diff
@@ -37,21 +34,93 @@ $ ./spin --help
 
 After you follow the [quickstart document](https://developer.fermyon.com/spin/quickstart/),
 you can follow the
-[Rust](https://developer.fermyon.com/spin/rust-components/) or [Go](https://developer.fermyon.com/spin/go-components/)
+[Rust](https://developer.fermyon.com/spin/rust-components/), [JavaScript](https://developer.fermyon.com/spin/javascript-components), [Python](https://developer.fermyon.com/spin/python-components), or [Go](https://developer.fermyon.com/spin/go-components/)
 language guides, and the [guide on configuring Spin applications](https://developer.fermyon.com/spin/configuration/).
 
-After you build your application, run it using Spin:
+Below is an example of using the `spin` CLI to create a new Spin Python application, then adding a JavaScript component:
```

Comment:
This sounds like creating a Python app and kind of linking in a JS module.  We don't really have good terminology for this - we loosely use "Python app" or "Rust app" for monolingual applications, but what you describe here is a multilingual app that has one component in Python and one in JS, and I don't know how to express that concisely.

My suggestion would be to show a single-component app, which gives more room to include the questions/responses from the `spin new` command, then link to the docs for more info.

Reply:
I revamped the example to make it simpler. It is now just a quick spin new, spin build, spin up.